assets/js/base/components/store-notices-container/style.scss

```diff
@@ -27,8 +27,9 @@
 
 .components-snackbar-list {
 	position: fixed;
-	bottom: 100px;
-	left: 100px;
+	bottom: 40px;
+	left: 16px;
+	width: auto;
 
 	@include breakpoint( "<600px" ) {
```

Comment:
I know that's not introduced in this PR so it can be fixed as a follow-up, but just leaving it here in case you want to fix it as well. I think we should use the same breakpoints we use for the _Cart_ and _Checkout_ blocks, which is `782px` instead of `600px`.